### ISSUE TITLE
X.default is not a constructor

### ISSUE THREAD

**Issue by ConduciveMammal** [Elkfox/Ajaxinate#45]
After loading in the package via Yarn, I'm given this console error: `Uncaught TypeError: i.default is not a constructor`.
Very similar to #30, however I'm not using Slate.

It appears to be an issue with the package itself as when I directly linked to the JS file on the demo page, it worked perfectly.

This is my code:

```js
import Ajaxinate from 'ajaxinate/dist/ajaxinate';

class InfiniteClick {
  constructor() {
    this.endlessClick();
  }

  endlessClick() {
    document.addEventListener("DOMContentLoaded", function() {
      const endlessScroll = new Ajaxinate({
        container: '#AjaxinateContainer',
        pagination: '#AjaxinatePagination'
      });
    });
  }
}

export default InfiniteClick;
```

**Comment by Cam** [comment]
We've got a new version coming soon @ConduciveMammal. The old version doesn't play well as a package.

**Comment by Cam** [comment]
Definitely related to #30

**Comment by ptboozer** [comment]
I had the same issue. Eagerly awaiting the new version. Thanks for rolling it out soon

**Comment by Cam** [comment]
I believe this was resolved with 5c975c4
